abc.templates/README.md

```diff
@@ -27,5 +27,8 @@ abc templates render \
 -input=service_name=<SERVICE_NAME> \
 -input=project_id=<PROJECT_ID> \
 -input=region=<REGION> \
+-input=github_owner_name=<GITHUB_ORG> \
 github.com/abcxyz/github-token-minter.git//abc.templates/deployments?ref=<TAG_OR_SHA>
 ```
+6. Add configurations in `deployments/configs/<github_owner_name>` for each repository.
+7. Run the deployment workflow.
```

Comment:
Can we do this in such a way that, after you merge these changes, the deployment will just go?

Reply:
Reworded, but yes that is the current process